src/app/client/src/app/service/popup-control.service.ts

```diff
@@ -8,11 +8,18 @@ export class PopupControlService {
 
   private popupSource = new BehaviorSubject(true);
   public checkPopupStatus = this.popupSource.asObservable();
-
+  private onboardPopupSubject = new BehaviorSubject<any>(null);
   constructor() { }
 
   changePopupStatus(value: boolean) {
     this.popupSource.next(value);
   }
 
+  setOnboardingData(formData: any) {
```

Comment:
@mithun30052001  add the comment description

Reply:
Comments added @princegupta1131